packages/compass-aggregations/src/modules/options.ts

```diff
@@ -0,0 +1,22 @@
+enum ActionTypes {
+  ToggleOption = 'aggregations/options/toggleOption',
+};
+
+type ToggleOptionAction = {
+  type: ActionTypes.ToggleOption;
+};
+
+type State = boolean;
+
+export const INITIAL_STATE: State = false;
+
+export default function reducer(state = INITIAL_STATE, action: ToggleOptionAction): State {
+  if (action.type === ActionTypes.ToggleOption) {
+    return !state;
+  }
+  return state;
+}
+
+export const toggleOptions = (): ToggleOptionAction => ({
+  type: ActionTypes.ToggleOption,
+});
```

Comment:
So before we used existing collation slice state and actions because collation was not part of our new toolbar, I remember us talking about that and this makes sense. Now collation input is completely encapsulated  in the new toolbar so why are we using redux for a state that can be local component state and doesn't need to be global?